### PR TITLE
Add smart_proxy_remote_execution_ssh (DEB)

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: plugins/smart_proxy_remote_execution_ssh/control [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Source: ruby-smart-proxy-remote-execution-ssh
+Section: ruby
+Priority: optional
+Maintainer: Stephen Benjamin <stephen@redhat.com>
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.3
+Homepage: https://github.com/theforeman/smart_proxy_remote-execution-ssh
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/control]: underscores, not hyphens

File: plugins/smart_proxy_remote_execution_ssh/control [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+XS-Ruby-Versions: all
+
+Package: ruby-smart-proxy-remote-execution-ssh
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.9.0~rc3), ruby-smart-proxy-dynflow
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/control]: ruby-smart-proxy-dynflow should be pinned to ~> 0.0.3

Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/control]: Also missing net-scp/net-ssh deps.

File: plugins/smart_proxy_remote_execution_ssh/copyright [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-smart-proxy-remote-execution-ssh
+Source: https://github.com/theforeman/smart_proxy_remote-execution-ssh
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/copyright]: ditto, hyphens

File: plugins/smart_proxy_remote_execution_ssh/copyright [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Files: *
+Copyright: 2012-2013 Red Hat Inc. <foreman-dev@googlegroups.com>
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/copyright]: ditto, 2015?

File: plugins/smart_proxy_remote_execution_ssh/control [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Source: ruby-smart-proxy-remote-execution-ssh
+Section: ruby
+Priority: optional
+Maintainer: Stephen Benjamin <stephen@redhat.com>
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.3
+Homepage: https://github.com/theforeman/smart_proxy_remote_execution-ssh
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/control]: Another hyphen!

File: plugins/smart_proxy_remote_execution_ssh/copyright [hunk 6]
```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-smart-proxy-remote-execution-ssh
+Source: https://github.com/theforeman/smart_proxy_remote_execution-ssh
```
Comment (domcleal) [inline plugins/smart_proxy_remote_execution_ssh/copyright]: and here

File: plugins/smart_proxy_remote_execution_ssh/control (hunk 5 above)
Comment (stbenjam) [inline plugins/smart_proxy_remote_execution_ssh/control]: At least the hyphens are consistent?

**Comment by domcleal** [comment]
Merged as 38a8a80, thanks @stbenjam.

**Pull request closed without merging**